src/coder/encode.js

```diff
@@ -29,7 +28,7 @@ class Encoder {
     const header = pool.slice(this._poolOffset, offset)
```

Comment:
We'll need to check where we're doing slices and look at using subarray instead, https://nodejs.org/api/buffer.html#buffer_buf_subarray_start_end. slice will cause a clone on TypedArrays, whereas with Buffer it will reference the same memory.

Reply:
This is now using `subarray` for the browser version.